### PR TITLE
Improve smokeping dashboard

### PR DESCRIPTION
**Pull request by nschermer**
I found the smokeping html output a bit dull and hard to format with css so I tried to improve the situation a bit.

The main changes are:

- Print an ul list menu instead of table.
- Add "htmltitle" config option to render graph title in html and not in image.
- Allow formatting of the graphs using divs to wrap it all up.
- Update sidebar filter and hierarchy selector, also to allow easier styling.
- Update rrdtool/smokeping logos (smaller, matching backgrounds).
- Make new basepage to showoff all above.

Things to improve:

- Improve mobile support (hide-able sidebar for example).
- Make background and border colours of the generated graphs more consistent. I personally prefer to have no borders and a white bg since css can be used to to style this (as the 3hr image in the screenshot below). Maybe a config option can be used for this, no idea... But a bit more consistency is always good ;).

The newly generated sidebar takes a bit more space (vertically) but is also easier on the eyes, maybe margins can be a bit smaller here, but I think for most users this is already a good start...

### CODE REVIEW

**Comment by nschermer** [comment]
And because we all love screenshots:
shot

**Comment by oetiker** [comment]
cool .. I like it!

**Comment by nschermer** [comment]
Nice, but give me some extra days for the final tweaks because there are still some cases where the generated html is not perfect or missing the panel class.

**Comment by oetiker** [comment]
just send another PR :)

File: lib/Smokeping.pm [hunk 1]
```diff
@@ -736,26 +735,27 @@ sub target_menu($$$$;$){
                  $menuextra = '';
              }
 
-          	 my $menuadd ="";
-		     $menuadd = "&nbsp;" x (20 - length($menu.$menuextra)) if length($menu.$menuextra) < 20;
-             $print .= qq{<tr><td class="$class" colspan="2">&nbsp;-&nbsp;<a class="$menuclass" HREF="$path$key$suffix">$menu</a>$menuextra$menuadd</td></tr>\n};
+          	$print .= qq{<li class="$class"><a class="$menuclass" href="$path$key$suffix">$menu</a>\n};
```
Comment (cab-umn) [inline lib/Smokeping.pm:738]: The menuextra appears to have been indavertently omitted during this refactor and no longer appears next to menu items.

Comment (oetiker) [reply lib/Smokeping.pm:738]: we have another big rewrite in the queue! #467

**Pull request merged**